### PR TITLE
Role: RDS Root CA Certificates

### PR DESCRIPTION
**Pull request by paddyo**
For anyone interested in using full on SSL when connecting to an RDS service. This role gets the certificate, adds it to a new truststore & imports the default Java truststore certificates into that one.

### CODE REVIEW

File: roles/aws-rds-root-cert-ca/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+truststore_password: changeit
```
Comment (AWare) [inline roles/aws-rds-root-cert-ca/defaults/main.yml]: Do we want to set defaults at all for this, it seems more dangerous than letting it fail if they're not set?

**Comment by sihil** [comment]
Have we thought about bundling with the the java roles? It makes sense to me that we should have java that just works. What are the pros and cons here? Have to maintain multiple versions for each java we support? This might be the case anyway as the truststore path might be different?

**Comment by paddyo** [comment]
Yeah apparently the truststore paths are pretty arbitrary apparently. Also most people won't be using these RDS certs because the VPC will do the job for them.

**Comment by sihil** [comment]
As in we don't consider necessary to ensure the link is secure within a VPC?

**Comment by paddyo** [comment]
Yes, I believe VPC is considered secure for a lot of people (not sure if that's correct or not). We are definitely putting an extra layer on identity though because of personal information.

**Pull request merged**